owncloudData/src/androidTest/java/com/owncloud/android/data/roommigrations/MigrationToDB36Test.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * ownCloud Android client application
+ *
+ * @author Fernando Sanz Velasco
+ * Copyright (C) 2021 ownCloud GmbH.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package com.owncloud.android.data.roommigrations
+
+import androidx.sqlite.db.SupportSQLiteDatabase
+import androidx.test.filters.SmallTest
+import com.owncloud.android.data.OwncloudDatabase
+import com.owncloud.android.data.ProviderMeta.ProviderTableMeta.CAPABILITIES_TABLE_NAME
+import com.owncloud.android.testutil.OC_CAPABILITY
+import org.junit.Assert
+import org.junit.Test
+
+@SmallTest
+class MigrationToDB36Test : MigrationTest() {
+
+    @Test
+    fun migrationFrom34to35_containsCorrectData() {
```

Comment:
Check typos in the file: `migrationFrom34to35` should be `migrationFrom35to36` 👍

Reply:
I forgot it. Good catch 😉